### PR TITLE
Crud dev

### PR DESCRIPTION
**Pull request by lfelberg**
Backend refactor - adding in server endpoints and starting on CRUD methods to mysql db

### CODE REVIEW

File: database/controller.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+module.exports.updateImage = (id) => {
+  // const queryString = 'SELECT imageUrl FROM images WHERE id = (?)';
+  // return db.query(queryString, [id]);
+};
+
+module.exports.deleteImage = (id) => {
+  // const queryString = 'SELECT imageUrl FROM images WHERE id = (?)';
+  // return db.query(queryString, [id]);
+};
+
```
Comment (CharlieAxelrod) [inline database/controller.js:22]: I know that we're not using these endpoints right now, but should they be commented out? Not a big deal, though.

Comment (CharlieAxelrod) [reply database/controller.js:22]: Also, all of these queries look the same?

File: database/controller.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+module.exports.getAllImages = (houseID) => {
+  const queryString = 'SELECT imageUrl FROM images WHERE houseID = (?)';
+  return db.query(queryString, [houseID]);
+};
```
Comment (CharlieAxelrod) [inline database/controller.js:26]: Wouldn't this be 'SELECT * ...' ?

File: server/home.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const db = require('../database/index.js');
+
+module.exports.getByHomeId = (req, res) => {
+  const id = req.params.home;
+  db.getAllImages(id, (err, results) => {
+    if (err) {
+      res.status(500).send(err);
+    } else {
+      res.status(200).send(results);
+    }
```
Comment (CharlieAxelrod) [inline server/home.js]: This looks great!

File: database/controller.js (hunk 2 above)
Comment (CharlieAxelrod) [reply database/controller.js:26]: Later you call this with an additional parameter (a callback function).

File: server/images.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+const db = require('../database/index.js');
+
+module.exports.get = (req, res) => {
+  const id = req.params.song;
```
Comment (CharlieAxelrod) [inline server/images.js]: What is song?

**Review by CharlieAxelrod — CHANGES_REQUESTED** [review]
Hey Lisa! Looks good overall, but there are a few things here and there that I'm confused about. Could you let me know?

File: database/controller.js (hunk 1 above)
Comment (lfelberg) [reply database/controller.js:22]: Flushed out these queries, to be more complete!

File: database/controller.js (hunk 2 above)
Comment (lfelberg) [reply database/controller.js:26]: This is for the houses endpoint. I don't think we need any information other than imageUrls, so `SELECT imageUrl` replaces `SELECT *`

Comment (lfelberg) [reply database/controller.js:26]: Fixed this function call in server routes

File: server/images.js (hunk 4 above)
Comment (lfelberg) [reply server/images.js]: Oops! lingering information/implementation from my old project!

**Comment by lfelberg** [comment]
Okay. I added the newest changes. Would you mind reviewing it again?

**Comment by CharlieAxelrod** [comment]
Looks good!

**Pull request merged**